x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_detector/detector_cards.tsx

```diff
@@ -31,6 +31,45 @@ export const CountCard: FC<CardProps> = ({ onClick, isSelected }) => (
             id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.description"
             defaultMessage="Look for anomalies in the event rate of a category."
           />
+
+          <EuiSpacer size="s" />
+
+          <FormattedMessage
+            id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.description2"
+            defaultMessage="Using Count is more appropriate when categorizing all messages."
+          />
+        </>
+      }
+      selectable={{ onClick, isSelected }}
+    />
+  </EuiFlexItem>
+);
+
+export const HighCountCard: FC<CardProps> = ({ onClick, isSelected }) => (
+  <EuiFlexItem>
+    <EuiCard
+      data-test-subj={`mlJobWizardCategorizationDetectorHighCountCard${
+        isSelected ? ' selected' : ''
+      }`}
+      title={i18n.translate(
+        'xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.highCountCard.title',
+        {
+          defaultMessage: 'High count',
+        }
+      )}
+      description={
+        <>
+          <FormattedMessage
+            id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.highCountCard.description"
+            defaultMessage="Look for unusually high counts of a category in the event rate."
+          />
+
+          <EuiSpacer size="s" />
+
+          <FormattedMessage
+            id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.highCountCard.description2"
+            defaultMessage="Using High Count is more appropriate when categorizing just error messages."
```

Comment:
Nit - I'd drop the word `just` here.

Reply:
Fixed by adopting @szabosteve 's suggestions